### DISCUSSION TITLE
Copy full path to clipboard

### DISCUSSION THREAD

**Discussion by as000fm** [mockoon/mockoon#1199] [Help]
I must be doing something wrong...

Even if I set a FQDN for the API URL in the settings of an environment, I end-up with localhost instead of the FQDN when I do a "Copy full path to clipboard" of a route ?!?

(Version 5.1.0)

As you can see, this is the setting for the API URL:

![image](https://github.com/mockoon/mockoon/assets/61089329/778d4d52-4d67-48f1-853e-ca4197d23c86)

And when I do a "Copy full path to clipboard" for a route...

![image](https://github.com/mockoon/mockoon/assets/61089329/dfddc4e7-a5c0-4f05-a3dc-098cbb7e7a41)

I end-up with **http://localhost:3001/Test** instead of **http://my.domain.net:3001/Test**

Is there a configuration in Mockoon to use the **API URL** instead of **localhost** when copying the full path to the clipboard of a route ?

Thanks.

**Comment by 255kb** [comment]
"localhost" is hardcoded in the method called when copying the full path 🙈
I will fix this for the next release!

**Comment by as000fm** [comment]
Thanks !!!
